ravenframework/Models/EnsembleModel.py

```diff
@@ -545,6 +553,19 @@ def submit(self,myInput,samplerType,jobHandler,**kwargs):
     ## works, we are unable to pass a member function as a job because the
     ## pp library loses track of what self is, so instead we call it from the
     ## class and pass self in as the first parameter
+    if  self.testj is None:
+      self.testj = copy.deepcopy(jobHandler)
+      self.testj.terminateAll()
```

Comment:
Hm, I think this should call a function in JobHandler. Something like:
```
self.testj = jobHandler.createClientJobHandler()
```
and JobHandler could have:
```
def createClientJobHandler(self):
    client = copy.deepcopy(self)
    client.terminateAll()
    return client
```
Possibly also the pollingThread should be created there as well.
Basically, I think JobHandler should be the one to handle the creating a semi-clone of itself.
Thanks.

Reply:
done